zmon-agent.py

```diff
@@ -171,6 +172,27 @@ def get_rds_instances(region, acc):
 
     return rds_instances
 
+def get_cassandra_instances(region):
+    cassandra_instances = []
+
+    try:
+        aws = boto.ec2.connect_to_region(region)
+        instances = aws.get_only_instances()
+
+        for i in instances:
+                if("Name" in i.tags and 'cassandra' in i.tags['Name'] and 'opscenter' not in i.tags['Name']):
+                    cassandraInstance = {'type' : 'cassandra', 'region': region, 'created_by':'agent'}
+                    cassandraInstance["host"] = i.private_ip_address.encode("utf-8")
+                    cassandraInstance["stackName"] = i.tags['StackName'].encode("utf-8")
+                    cassandraInstance["stackVersion"] = i.tags['StackVersion'].encode("utf-8")
+                    cassandraInstance["instanceType"] = i.instance_type.encode("utf-8")
```

Comment:
you also need the infrastructure account, otherwise our mapping does not work.

I would also include the docker image in "source", enabling version checks.

basically it may just be added to the instance iterator and add them to "cassandra_instances" if "source" contains "stups-cassandra" for now